From f960e63014809beae40113d2c82ed06f1e4c8273 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 09 Nov 2021 13:50:10 +0200
Subject: [PATCH] [SW-0] gaudi2: pll: raise the pll reset time

The spec says we need to wait at least 1us between applying the
coefficients and enabling (unresetting) the PLL.
Raise this time to 5us as 1us is the minimum allowed.

Change-Id: I10be8dff7b8b043844b45cd4fd27ae3a6982f341
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/pll.c b/arch/gaudi2/common/pll.c
index 77ad853..f906a78 100644
--- a/arch/gaudi2/common/pll.c
+++ b/arch/gaudi2/common/pll.c
@@ -75,7 +75,7 @@
 #define GAUDI2_PLL_POSTDIV1_SHIFT	16
 #define GAUDI2_PLL_POSTDIV2_SHIFT	20
 #define GAUDI2_PLL_REFDIV_SHIFT		24
-#define GAUDI2_PLL_MAX_WAIT_TIME		1
+#define GAUDI2_PLL_MAX_WAIT_TIME	5
 
 void hl_pll_cfg(struct hl_pll_cfg *cfg)
 {
